github/PullRequest.py

```diff
@@ -284,6 +284,14 @@ def state(self):
         return self._state.value
 
     @property
+    def squash(self):
```

Comment:
I do not see a PullRequest "squash" property in https://developer.github.com/v3/pulls/ or in the responses I see from github today. When does a PullRequest receive a "squash" property? Or is this wrong/outdated?